src/js/utils/ui.js

```diff
@@ -129,7 +129,7 @@ const UI = function (elem, options) {
 
     // overHandler and outHandler not assigned in touch situations
     function overHandler(evt) {
-        if (evt.pointerType !== 'touch') {
+        if (evt.pointerType !== 'touch' && evt.fromElement.classList[0].indexOf('jw-nextup') === -1) {
```

Comment:
Ui.js is a generic util. Custom logic for nextup should not go into it. What is listening to `OVER` that causes controls to toggle? This logic should be handled there - likely in view.js or controls.js or a submodule of controls.